esx_service/vmdk_ops.py

```diff
@@ -1297,16 +1311,20 @@ def err(string):
     return {u'Error': string}
 
 
-def disk_detach(vmdk_path, vm):
-    """detach disk (by full path) from a vm amd return None or err(msg)"""
+def disk_detach(full_vol_name, vm):
```

Comment:
But the content of param 'full_vol_name' is qualified volume name in format volume@datastore
Calling it full_vmdk_path would be confusing.
I will update the param comment though